src/Base64KeyLoader.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Acquia\Hmac;
+
+/**
+ * Base64KeyLoader class for loading keys and Base64 encoding them.
+ *
+ * These keys can be later used for signing and verifying requests.
+ */
+class Base64KeyLoader implements KeyLoaderInterface
```

Comment:
Since this is just a convenience wrapper around the regular `KeyLoader`, I think I'd prefer this extend it and simply override `KeyLoader::load()`.